src/Compilers/CSharp/Test/Emit2/Semantics/InlineArrayTests.cs

```diff
@@ -12822,9 +12852,24 @@
         _element0 = 1;
     }
 }
+
+[System.Runtime.CompilerServices.InlineArray(1)]
+public ref struct Buffer1Ref
+{
+    public ref int _element2;
```

Comment:
>_element2 [](http://example.com/codeflow?start=19&length=9)

Consider renaming to `_element0` so it's clear the name is not important.

Reply:
> Consider renaming to _element0 so it's clear the name is not important.

The name is distinct so that it is easy to see what line causes an error. I will keep it as is.